.github/cmake.yml

```diff
@@ -0,0 +1,105 @@
+#******************************************************************************
+#*Copyright (C) 2023 by Salvador Z                                            *
+#*                                                                            *
+#*****************************************************************************/
+#*
+#*@author Salvador Z
+#*@brief GitHub workflow to configure an automated process to run one or more jobs
+#*
+# Template for all jobs
+name: GITHUB-CI
+
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches:
+      - develop
+    types:
+      - closed
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Debug
```

Comment:
* Change to Release and for Code coverage perform a clean and build with `BUILD_TYPE`=`Debug`